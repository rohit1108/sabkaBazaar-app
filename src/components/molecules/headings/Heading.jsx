const Heading = ({heading,className,children}) =>{
    return (
        <div className={className}>
            <h1>{heading}</h1>
            <p>{children}</p>
        </div>
    )
 }
export default Heading

