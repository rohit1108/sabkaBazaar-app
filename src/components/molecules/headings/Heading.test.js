import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
//import { AppWideContext } from '../../contexts/AppWideContext';
import ReactDom from "react-dom";
import Heading from './Heading';
import { GlobalContext } from '../../../context/GlobalContext';

describe("Header Component", () => {

    it("should render the component properly", () => {
        const headerEl = document.createElement("header");
        ReactDom.render(<GlobalContext.Provider value={{ cartItems: { count: 0 } }}><BrowserRouter><Heading/></BrowserRouter></GlobalContext.Provider>, headerEl); 
    });

    it('should have the right header component snapshot', () => {
        const tree = renderer.create(<GlobalContext.Provider value={{ cartItems: { count: 0 } }}><BrowserRouter><Heading /></BrowserRouter></GlobalContext.Provider>).toJSON();
        expect(tree).toMatchSnapshot();
    })
});