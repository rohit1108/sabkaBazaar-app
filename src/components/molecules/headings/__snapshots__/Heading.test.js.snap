// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should have the right header component snapshot 1`] = `
<div>
  <h1 />
  <p />
</div>
`;
